lighthouse-core/audits/time-to-interactive.js

```diff
@@ -29,6 +29,7 @@ class TTIMetric extends Audit {
       category: 'Performance',
       name: 'time-to-interactive',
       description: 'Time To Interactive (alpha)',
+      helpText: 'Page load performance is a key factor in how users perceive your app\'s performance. Time to Interactive is a metric that identifies the time at which your app appears to be ready enough to interactive with. <a href="https://developers.google.com/web/tools/lighthouse/audits/time-to-interactive" target="lighthouse" rel="noopener noreferrer">Learn more</a>.',
```

Comment:
yea, I think for these just hope right into what it is and where to learn more. Good for keeping the report minimal.